scripts/aptly-repo-from-debs.sh

```diff
@@ -64,9 +64,7 @@ done
 # Download the delphix upgrade verification debian package, stored in the
 # combined-packages bundle.
 #
-AWS_S3_URI_COMBINED_PACKAGES=$(resolve_s3_uri \
-	"$AWS_S3_URI_COMBINED_PACKAGES" \
-	"linux-pkg/${UPSTREAM_BRANCH}/combine-packages/post-push/latest")
+AWS_S3_URI_COMBINED_PACKAGES=$(resolve_s3_uri "$AWS_S3_URI_COMBINED_PACKAGES")
```

Comment:
I'm confused, what is this line doing? seems like it's not actually doing anything?

What happens when `AWS_S3_URI_COMBINED_PACKAGES` is empty?

Reply:
oh.. nvm, I hadn't seen the changes to `resolve_s3_uri` yet..